torch/_refs/__init__.py

```diff
@@ -4731,7 +4846,6 @@ def unsqueeze(a: TensorLikeType, dim: int) -> TensorLikeType:
 # Tensor.view(a, b, c) or Tensor.view((a, b, c)) Function call torch.view
 # doesn't support unpacked shapes
 # TODO: Turn this into a decomposition (currently fails on reshape meta tests)
-@register_decomposition(aten.view.default)
```

Comment:
we only need meta this was just used for fake tensor tracing. 
but this is not dynamic shapes friendly and not worth the complixity if its just for fake tensor tracing and the opts
not recorded in the graph lol